front/ticket.form.php

```diff
@@ -32,10 +32,7 @@
 
 include('../../../inc/includes.php');
 
-// Same right check as in PluginEscaladeTicket::addToTimeline()
-Session::checkRight(Ticket::$rightname, Ticket::READALL)
-    && Session::checkRight(Ticket::$rightname, Ticket::READASSIGN)
-    && Session::checkRight(Ticket::$rightname, Ticket::CREATE);
+Session::checkLoginUser();
```

Comment:
It seems a bit dangerous to permit to any connected user to escalate any ticket to a group.

Reply:
Corrected